### DISCUSSION TITLE
Launcher settings "app"

### DISCUSSION THREAD

**Discussion by undrwater** [otakuhqz/Omega#192] [Ideas]
Somehow I disabled the long press gesture to get to settings. Haven't figured out how to change that yet, but maybe having an app will help users get out of a similar jam.

**Comment by saulhdev** [comment]
There are a few ways to open the Launcher settings.
1. ACTIONS
Long press on home screen, select widgets, in the Neo App, drag actions to the Home screen, and choose Launcher Settings. 

2. DASH
If dash is enabled through gestures, you can click on the launcher settings Icon.

3. DEVICE SETTINGS
Open the device settings > Apps > Neo Launcher 
then click on In-app notification settings.

You should use the option 3.

**Reply by undrwater** [reply]
Thanks! Very helpful, but maybe not so intuitive?

Maybe a warning when turning off the long press gesture?
